docs/source/user_guide/enhanced-script-support.md

```diff
@@ -85,9 +85,20 @@ Another option is to install the R language server through the [CRAN](https://cr
 $ R -e install.packages("languageserver")
 ```
 
-## Ability to execute a Python and R script as a pipeline
+## Scala script execution support
 
-Elyra allows the execution of Python and R scripts as a pipeline or batch job in remote cloud environments. This feature leverages the AI pipelines feature and requires access to either a Kubeflow Pipelines or Apache Airflow deployment via a [runtime configuration](../user_guide/runtime-conf).
+In the JupyterLab Launcher, click the `Scala Editor` icon to create a new Scala script and open the Scala Editor.
+
+![Open Scala Editor](../images/launcher-scala-editor.png)
+
+When used in conjunction with `Jupyter Enterprise Gateway`, the dropdown in the editor's toolbar will be populated with more kernel options,
+allowing users to run their scripts with remote kernels with more specialized resources.
+
+To run your script locally, select the `Spark/Scala Toree Kernel` option in the dropdown menu, and click the `Run` icon.
+
+## Ability to execute a Python, R and Scala script as a pipeline
```

Comment:
Will this feature be added to this PR?

Reply:
This is where we add the submit button to other script editors from the pipeline editor package https://github.com/elyra-ai/elyra/blob/main/packages/pipeline-editor/src/index.ts#L336

Reply:
I believe there should be additional places that would require updates because file type verification is performed if one tries to drag and drop a file onto the Pipeline Editor canvas.

Reply:
As discussed in scrum, will do the `run as pipeline` in another pr

Reply:
Are you planning to deliver the PR in 3.11? If not, we should undo some of the doc changes to avoid the impression that Scala scripts are supported in pipelines.

Reply:
ive removed the references to scala in this section